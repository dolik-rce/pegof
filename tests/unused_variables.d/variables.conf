optimize unused-variable
input unused_variables.d/variables.peg
header never
