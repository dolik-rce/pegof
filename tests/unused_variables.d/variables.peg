%value "char"

X <- a:A B { printf("%c\n", a); }
A <- .* { $$ = 'a'; }
B <- "B" a:A { $$ = 'blah'; }
