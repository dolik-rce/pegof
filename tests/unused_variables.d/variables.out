%value "char"

X <- a:A (
    B
    / C
) { printf("%c\n", a); }

A <- .* { $$ = 'a'; }

B <- "B" A { $$ = 'blah'; }

C <- "C" (A .*) { $$ = 'blah'; }
